init/main.c

```diff
@@ -27,9 +27,16 @@ static const char *TABLE_LIST[] = {
 		"FOREIGN KEY (session_id) REFERENCES session(rowid),"
 		"PRIMARY KEY (session_id, user)"
 	")",
-	"CREATE TABLE type ("
-		"name TEXT PRIMARY KEY NOT NULL"
+	"CREATE TABLE type_log ("
+		"name TEXT NOT NULL,"
+		"added DATETIME NOT NULL,"
+		"removed DATETIME,"
+
```

Comment:
Why empty line?

Reply:
Think it is nice to separate constraints and columns